src/couch_epi_functions_gen.erl

```diff
@@ -309,6 +309,11 @@ remove_from_definitions(Defs, Source) ->
             {[], Defs}
     end.
 
+-spec modules(Handle :: atom(), Function :: atom(), Arity :: pos_integer()) ->
+    boolean().
```

Comment:
Return type is not boolean() here - it's a list which returned by [`couch_epi_codegen:scan/1`](https://github.com/apache/couchdb-couch-epi/blob/master/src/couch_epi_codegen.erl#L36) which is called by [`providers/4`](https://github.com/iilyak/couchdb-couch-epi/blob/add-is_configured/src/couch_epi_functions_gen.erl#L295-L302) (and /2,3 as well).